aospy/automate.py

```diff
@@ -257,25 +255,37 @@ def _compute_or_skip_on_error(calc, compute_kwargs):
         return None
 
 
-def _exec_calcs(calcs, parallelize=False, **compute_kwargs):
+def _exec_calcs(calcs, parallelize=False, client=None, **compute_kwargs):
     """Execute the given calculations.
 
     Parameters
     ----------
     calcs : Sequence of ``aospy.Calc`` objects
     parallelize : bool, default False
         Whether to submit the calculations in parallel or not
+    client : distributed.Client or None
+        The distributed Client used if parallelize is set to True; if None
+        a distributed LocalCluster is used.
     compute_kwargs : dict of keyword arguments passed to ``Calc.compute``
 
     Returns
     -------
     A list of the values returned by each Calc object that was executed.
     """
     if parallelize:
-        pool = multiprocess.Pool()
-        return pool.map(lambda calc:
-                        _compute_or_skip_on_error(calc, compute_kwargs),
-                        calcs)
+        f = lambda calc: _compute_or_skip_on_error(calc, compute_kwargs)
+        if client is None:
```

Comment:
Technically PEP8 is to use lambda only for inline functions when passed as arguments, e.g. `foo(bar=lambda x:x**2)`.  So use `def` here.